content/en/blog/2022/2022-05-25-contextual-logging.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: blog
+title: "Contextual Logging in Kubernetes 1.24"
+date: 2022-05-25
+slug: contextual-logging
+---
+
+ **Authors:** Patrick Ohly (Intel)
+
+The [Structured Logging Working
+Group](https://github.com/kubernetes/community/blob/master/wg-structured-logging/README.md)
+has added new capabilities to the logging infrastructure in Kubernetes
+1.24. This blog post explains how developers can take advantage of those to
+make log output more useful.
```

Comment:
suggestion: put a call for "and you can join us and help" here too, not just at the bottom

Reply:
Makes sense, will change it.